lib/cyclonedx/cyclonedx_report.rb

```diff
@@ -0,0 +1,55 @@
+require 'securerandom'
+
+Dir.entries(File.expand_path('./', __dir__)).sort.each do |filename|
+  next unless /_cyclonedx.rb\z/.match?(filename) && !filename.eql?('base_cyclonedx.rb')
```

Comment:
where will you require `base_cyclonedx.rb`?  Probably need to ensure that gets required before the other files.

Reply:
Good catch, added